puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/ansible.groovy

```diff
@@ -1,32 +1,20 @@
-def runPlaybook(body) {
+def runPlaybook(playbook, inventory, extraVars = [], options = []) {
+    def command = [
+        "ansible-playbook",
+        "-i ${inventory}",
+        playbook
+    ]
 
-    def config = [:]
-    body.resolveStrategy = Closure.DELEGATE_FIRST
-    body.delegate = config
-    body()
-
-    wrap([$class: 'AnsiColorBuildWrapper', colorMapName: "xterm"]) {
-
-        def extraVars = [:]
-        def defaultVars = [:]
-        def inventory = config.inventory
-        def tags = config.tags
-        def limit = config.limit
-
-        if (config.extraVars) {
-            extraVars = defaultVars + config.extraVars
-        } else {
-            extraVars = defaultVars
-        }
+    if (options) {
+        command += options
+    }
 
-        ansiblePlaybook(
```

Comment:
any reason this can no longer be used?

Reply:
No, but this way feels easier and faster way of using this instead of the class command.

Reply:
Faster in which way? It feels like this function is now reinventing this class command.

Reply:
I made a poor assumption, it's not for any of those reasons. It turns out that ci.theforeman.org has the plugin that allows `ansiblePlaybook` to be run. ci.cenos.org does not have that plugin and this was the way @ehelms got around not having it.